test/notifications/display_test.dart

```diff
@@ -131,6 +131,59 @@ void main() {
             NotificationChannelManager.kVibrationPattern)
       ;
     });
+
+    test('channel is not recreated if one with same id already exists', () async {
+      await NotificationChannelManager.ensureChannel();
+      check(testBinding.androidNotificationHost.takeChannelMethodCallLogs())
+        .deepEquals(['getNotificationChannels', 'createNotificationChannel']);
+
+      await NotificationChannelManager.ensureChannel();
+      check(testBinding.androidNotificationHost.takeChannelMethodCallLogs())
+        .deepEquals(['getNotificationChannels']);
```

Comment:
I'm generally not a fan of having "get" calls in a log that a test checks like this — it feels too far into testing the implementation details (cf https://zulip.readthedocs.io/en/stable/testing/philosophy.html#integration-testing-or-unit-testing ).

(We do routinely include the equivalent in our API tests — we check what HTTP requests the app made to the simulated server. But because these queries are local to the device, I'd like to try to treat them more as internal.)

Can we leave these out of the log, and have the tests just look at what the code does with the information? I.e., the `create` and `delete` calls.